app/src/test/java/uk/ac/ebi/atlas/experimentpage/markergenes/MarkerGenesDaoIT.java

```diff
@@ -111,4 +111,18 @@ void shouldFetchOnlyInferredCellTypeMarkerGenes() {
         return jdbcTestUtils.fetchKsFromCellGroups(EXPERIMENT_ACCESSION_WITH_MARKER_GENES)
                 .stream().map(Object::toString);
     }
+
+    @Test
+    void shouldGetCellTypesWithMarkerGenesGivenCellGroup() {
+        var cellTypesWithMarkerGenes = subject.getCellTypesWithMarkerGenes("E-MTAB-5061", "Inferred cell type - ontology labels");
+        assertThat(cellTypesWithMarkerGenes).isNotEmpty();
+
+    }
+
+    @Test
+    void shouldGetCellTypeMarkerGenesHeatmapData() {
```

Comment:
test method name would be some thing like this for the readability- `shouldGetMarkerGenesHeatmapDataForTheGivenCellGroupAndCellType()`